src/kata1.fizzBuzz.js

```diff
@@ -1,3 +1,18 @@
-const fizzBuzz = (number) => {};
+const fizzBuzz = (number) => {
```

Comment:
Indentation is different in this file, the rest of your work uses 2 spaces so you should do the same here